pkg/api/azenvtypes.go

```diff
@@ -188,15 +188,15 @@ var (
 		ImageOffer:     "aks-windows",
 		ImageSku:       "2019-datacenter-core-smalldisk-2011",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.1554.201109",
+		ImageVersion:   "17763.1577.201112",
 	}
 
 	// WindowsServer2019OSImageConfig is the 'vanilla' Windows Server 2019 image
 	WindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "WindowsServer",
 		ImageSku:       "2019-Datacenter-Core-with-Containers-smalldisk",
 		ImagePublisher: "MicrosoftWindowsServer",
-		ImageVersion:   "17763.1397.2008070242",
+		ImageVersion:   "17763.1577.2011031610",
```

Comment:
@jsturtevant did you build `17763.1577.201112` based on `17763.1577.2011031610`?

Reply:
FYI @yizhang4321